test/smart_city/dataset/business_test.exs

```diff
@@ -30,12 +31,16 @@ defmodule SmartCity.Dataset.BusinessTest do
       assert actual.rights == nil
     end
 
-    test "allows blank modified dates", %{message: biz} do
+    test "allows blank modified dates and does not log a message", %{message: biz} do
```

Comment:
I understand that you modified this test to validate modifying the production code, but do we really need the negative test here?